agent/engine/execcmd/manager_init_task_windows.go

```diff
@@ -16,18 +16,209 @@
 package execcmd
 
 import (
```

Comment:
Should unit tests and integ tests be included for these changes?

Reply:
yes, I will add them in now